modules/ROOT/pages/persistence-gateway.adoc

```diff
@@ -0,0 +1,122 @@
+= Persistence Gateway
+
+To store state across applications, Anypoint Runtime Fabric provides the Persistence Gateway. Persistene Gateway enables applications deployed to a Mule runtime engine (Mule) to store and share data across batch processes, Mule components and applications.
+
+[NOTE]
+====
+Persistence Gateway is supported on Anypoint Runtime Fabric, version 1.9 or later and Mule runtime engine, version 4.2.1 or later.
+====
+
+== Use Persistence Gateway in a Mule Application
+
+After Persistence Gateway is enabled, it is available when deploying to Mule runtime engine, version 4.2.1 or later. Mule applications use the Object Store REST API to connect to the Persistence Gateway. This enables you to deploy to both Anypoint Runtime Fabric and CloudHub without having to modify your Mule application. See xref:deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric] for more information.
+
+When an application using persistent object storage is deleted, Runtime Fabric deletes all information stored in the Persistence Gateway.
+
+== Persistent Gateway Limits
+
+The following table lists the limits on the data stored by the Persistence Gateway:
+
+[cols="2*a"]
+|===
+| Maximum TTL. This is the amount of time data is stored in the Persistence Gateway  | 30 days
+| Maximum key length | 256 characters
+| Maximum value length | 10 MB
+| Maximum partitions per application | 10
+| Maximum keys per partition | 1000
+|===
+
+
+[Configure-Persistence-Gateway]
+== Configure Persistence Gateway
+
+To enable Persistence Gateway, you must create a Kubernetes custom resource that allows the cluster to connect to your persistence data store.
+
+=== Prerequisties
+
+Before enabling Persistence Gateway, ensure that you have the following:
+
+* A data source to contain the data stored by Persistence Gateway. This data source must be compatible with PostgresSQL 9 and above.
+* A Kubernetes secret containing the network information and credentials to connect to the data store.
+
+=== Create a Custom Resource and Secret
+
+. Create a Kubernetes custom resource for your data store.
+.. Copy the custom resource template from <<Kubernetes Custom Resource Template>> to a file called `custom-resource.yaml`.
+.. Modify the fields of the template as required for your environment.
+.. Run `kubectl apply -f custom-resource.yaml` to add the custom resource to your cluster.
+. Create a Kubernetes secret.
```

Comment:
This should be step 1 followed by `Create a Kubernetes custom resource for your data store.`

Reply:
Done!